aiida/manage/configuration/migrations/migrations.py

```diff
@@ -199,8 +199,8 @@ def downgrade(self, config: ConfigType) -> None:
                 config['options'][current] = config['options'].pop(new)
 
 
-class AbstractStorage(SingleMigration):
-    """Move the storage configuration under a top-level "storage" key.
+class AbstractStorageAndProcess(SingleMigration):
+    """Move the storage config under a top-level "storage" key and rabbitmq config under "processing".
```

Comment:
This is not actually true I think, although I think that would actually be ideal:
```python
{
    'storage': {
        'backend': 'string',
        'config': {}
    },
    'process_control': {
        'backend': 'string',
        'config': {}
    }
}
```
Think that is the clearest.

But at least if you keep the current layout, then please update the docstring.

Reply:
No I think that seems reasonable 👍

Reply:
done; don't say I never give you anything 😉